docs/Catalog/Packaging.md

```diff
@@ -0,0 +1,226 @@
+# Packaging Engine
+
+## Overview
+
+The Quilt Packaging Engine in the Quilt Platform allows administrators and
+developers to automate the process of creating Quilt packages from data stored
+in Amazon S3. It serves as a key component in Quilt's SDMS (Scientific Data
+Management System) strategy, enabling automated data ingestion and
+standardization. It consists of:
+
+1. Admin Settings GUI to enable package creation based on notifications from:
+   1. AWS Health Omics
+   2. Nextflow workflows using  `nf-prov`'s WRROC ([Workflow Run
+      RO-Crate](https://www.researchobject.org/workflow-run-crate/)) format
+2. SQS queue that will process package descriptions
+3. Documentation for creating custom EventBridge rules to invoke that queue
+
+## Admin Settings
+
+The simplest way to enable package creation is through the Admin Settings GUI,
+which supports the following built-in event sources:
+
+![Admin Settings](../imgs/package-admin-gui.png)
+
+### AWS Health Omics
+
+When enabled, this will create a package from the `runOutputUri` provided in a
+`aws.omics` completion event. For example, if the `runOutputUri` is
+`s3://quilt-example/omics-quilt/3395667`, the package will be created in that
+same bucket with the name `omics-quilt/3395667`.
+
+### Workflow Run RO-Crate
+
+When enabled, this will create a package when indexing any folder containing an
+`ro-crate-manifest.json`.  Indexing happens when the bucket is added to the
+stack, or when a folder is written to a bucket already in the stack.
+
+[RO-Crate](https://www.researchobject.org/ro-crate/) is a metadata standard for
+describing research data.  The Workflow Run working group adds three additional
+profiles, which are supported in the latest versions of
+[nf-prov](https://github.com/nextflow-io/nf-prov). You will need to explicitly
+configure `nf-prov` to use `wrroc`, by using a `nextflow.config` file [like
+this](https://github.com/famosab/wrrocmetatest):
+
+```groovy
+plugins {
+ id 'nf-prov@1.4.0'
+}
+
+prov {
+ enabled = true
+ formats {
+   wrroc {
+     file = "${params.outdir}/ro-crate-metadata.json"
+     overwrite = true
+     agent {
+       name = "John Doe"
+       orcid = "https://orcid.org/0000-0000-0000-0000"
+     }
+      license = "https://spdx.org/licenses/MIT"
+      profile = "provenance_run_crate"
+   }
+ }
+}
+```
+
+Note that Research Objects identify people using an ORCID iD, which anyone can
+get for free at [the ORCID website](https://orcid.org/).
+
+The package will be created in the same bucket as the `outdir`, with the package
+name inferred from the S3 key. For example, if the key is
+`my/s3/folder/ro-crate-manifest.json`, the package name will be `my_s3/folder`.
+
+## SQS Message Processing
+
+The primary interface to the Packaging Engine is through an SQS queue in the
```

Comment:
In order for this to make sense we need an overview of the components of the packaging engine. Right now you're assuming that the reader understands what the packaging engine is, and why the SQS queue is used. An overview of how the packaging engine is composed might be helpful (in a section above).